windows/devices/util/include/windows/devices/DeviceHandle.hxx

```diff
@@ -82,6 +83,7 @@ OpenDriverHandleShared(wil::shared_hfile &driverHandle, const char *deviceName,
 HRESULT
 OpenDriverHandleUnique(wil::unique_hfile &driverHandle, const char *deviceName, bool isOverlapped = false);
 
+const std::string NotificationHandleNamespaceString = "\\Notifications";

```

Comment:
This should really be published by the driver DDI in `UwbCxLrpDevice.h`. Even for now though, this file is for generic Windows device interaction, whereas the `uwbcx-driver` project is for UwbCx driver-specific information. So, this should move there. You can put it under 
[`windows/drivers/uwb/cx/include/windows/devices/uwb/UwbCxDdrLrp.hxx`](https://github.com/microsoft/nearobject-framework/blob/develop/windows/drivers/uwb/cx/include/windows/devices/uwb/UwbCxDdiLrp.hxx). Probably under a namespace of the same,

```
namespace windows::drivers::uwbcx
{
constexpr auto UwbNotificationNamespace "windows::drivers::uwbcx";

} // [namespace windows::devices::uwb](windows::drivers::uwbcx)

```